icu4c/source/common/unicode/localematcher.h

```diff
@@ -461,7 +460,7 @@ class U_COMMON_API LocaleMatcher : public UMemory {
          *
          * @param demotion the demotion per desired locale to set.
          * @return this Builder object
-         * @draft ICU 65
+         * @stable ICU 65
          */
         Builder &setDemotionPerDesiredLocale(ULocMatchDemotion demotion);
 
```

Comment:
Please add a new ifndef U_HIDE_DRAFT_API guard around the following setDirection() which is draft ICU 67.

Reply:
Sorry about that.  Hopefully fixed now.